aptos-move/framework/cached-packages/src/aptos_framework_sdk_builder.rs

```diff
@@ -3188,7 +3188,7 @@ pub fn delegation_pool_set_delegated_voter(new_voter: AccountAddress) -> Transac
         ),
         ident_str!("set_delegated_voter").to_owned(),
         vec![],
-        vec![bcs::to_bytes(&new_voter).unwrap()],
+        vec![bcs::to_bytes(&_new_voter).unwrap()],
```

Comment:
interesting didn't know you could continue to use `_` fields

Reply:
Yeah, let's see if it passes CI, otherwise you may need to leave the name

Reply:
Well leaving the name on the move side will generate warnings (unused variables) there